.github/workflows/build.yml

```diff
@@ -221,13 +221,13 @@ jobs:
           sed -i "s/appVersion: .*/appVersion: ${{steps.get-current-tag.outputs.tag }}/g" ./charts/casdoor/Chart.yaml
 
           # increase the patch version of the chart
-          currentChartVersion=$(cat ./charts/casdoor/Chart.yaml | grep version | awk '{print $2}')
+          currentChartVersion=$(cat ./charts/casdoor/Chart.yaml | grep ^version | awk '{print $2}')
```

Comment:
This sets currentChartVersion  to `Application` because [master](https://github.com/casdoor/casdoor-helm/blob/master/charts/casdoor/Chart.yaml#L18) is set to `Application This to 0.4.1`. You need to first manually correct [version](https://github.com/casdoor/casdoor-helm/blob/master/charts/casdoor/Chart.yaml#L18) and then merge in this change

Reply:
@sp71 I didn't see the version here: https://github.com/casdoor/casdoor-helm/blob/master/charts/casdoor/Chart.yaml#L18 got fixed

Reply:
Merged: https://github.com/casdoor/casdoor-helm/pull/4